helix-term/src/ui/picker.rs

```diff
@@ -293,7 +293,7 @@ impl<T: 'static> Component for Picker<T> {
                 inner.x + 3,
                 inner.y + 2 + i as u16,
                 (self.format_fn)(option),
-                inner.width as usize - 1,
+                inner.width as usize - 3, // account for the " > "
```

Comment:
Should be a `.saturating_sub` so we don't panic if the width is really small

Reply:
Done 👍🏾